examples/increment-counter/src/main.rs

```diff
@@ -9,117 +9,35 @@
-
-    // binary representation of the key used to store the counter value
-    let storage_key: StorageKey = [
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-        102, 111, 111,
-    ];
-    let mut buffer = [0u8; READ_BUFFER_LEN];
-
-    unsafe {
-        // Read from storage
-        match get(storage_key.as_ptr() as u32, buffer.as_mut_ptr() as u32) {
-            0 => (),
-            err => return err,
-        }
-
-        // Decode counter structure
-        let mut counter = if buffer == [0u8; READ_BUFFER_LEN] {
-            // Default value if zero
-            Counter::default()
-        } else {
-            Counter::decode(&mut &buffer[..]).unwrap_or(Counter::default())
-        };
-
-        // Increment by value
-        let new_value = counter.counter.saturating_add(increment);
-        call_print(&format!(
-            "incrementing counter: {} + {} -> {}",
-            counter.counter, increment, new_value
-        ));
-        counter.counter = new_value;
-
-        // Encode counter structure
-        buffer.fill(0);
-        for (pos, elem) in Counter::encode(&counter).iter().enumerate() {
-            buffer[pos] = *elem
-        }
+#[no_mangle]
+#[polkavm_derive::polkavm_export]
+pub extern "C" fn call() {
+    // Accept increment value from user input. Should be 4 bytes (e.g., `0x12345678`) or `ContractTrapped` error occurs.
+    input!(increment: u32, );
 
-        // Write to storage
-        match set(storage_key.as_ptr() as u32, buffer.as_mut_ptr() as u32) {
-            0 => (),
-            err => return err,
-        }
-    }
+    // Read the current value from storage.
+    unwrap_output!(raw, [0u8; 4], api::get_storage, StorageFlags::empty(), &KEY);
+    let old = u32::decode(&mut &raw[..]).unwrap();
```

Comment:
`raw` is a keyword in Rust:
https://doc.rust-lang.org/reference/keywords.html#r-lex.keywords.weak.intro
```suggestion
    unwrap_output!(raw_data, [0u8; 4], api::get_storage, StorageFlags::empty(), &KEY);
    let old = u32::decode(&mut &raw_data[..]).unwrap();
```